docker/install/build_flashinfer_ep_pytorch.sh

```
@@ -0,0 +1,52 @@
+echo "== base python / torch / cuda =="
+python --version
+python -c "import torch; print('torch', torch.__version__, 'cuda', torch.version.cuda)"
+nvcc --version | grep release || true
+
+echo "== pin NCCL-EP runtime wheels to ep_bench's verified set =="
+# PIP_CONSTRAINT= overrides the NVIDIA base image's constraint file (which pins
+# nvidia-nccl-cu13 to torch's 2.30.4) so we install the 2.30.7 that nccl4py 0.3.1's
+# libnccl_ep.so expects. --no-deps on the NCCL wheels keeps the base torch intact;
+# nccl.ep additionally imports cuda.core / cuda.bindings, installed explicitly at
+# ep_bench's exact versions (cuda-core 1.0.1, cuda-bindings 13.2.0).
+PIP_CONSTRAINT= pip install --no-cache-dir --no-deps \
```

Shellcheck (0.11.0)

[warning] Remove space after = if trying to assign a value (for empty string, use var='' ... ).

(SC1007)